Realm/Realm.modulemap

```diff
@@ -9,14 +9,14 @@ framework module Realm {
         header "RLMApp_Private.h"
         header "RLMArray_Private.h"
         header "RLMCollection_Private.h"
-        header "RLMObject_Private.h"
-        header "RLMObjectBase_Dynamic.h"
         header "RLMObjectBase_Private.h"
         header "RLMObjectSchema_Private.h"
         header "RLMObjectStore.h"
+        header "RLMObjectBase_Dynamic.h"
```

Comment:
why have these headers moved position?

Reply:
I added some things, sorted the list, then removed the new things as part of reducing this branch to just the changes to existing functionality, leaving the list unchanged other than being sorted.